pinot-common/src/main/java/com/linkedin/pinot/common/utils/SegmentName.java

```diff
@@ -21,6 +21,15 @@
   public static final String REALTIME_SUFFIX = "_REALTIME";
   public static final int REALTIME_SUFFIX_LENGTH = REALTIME_SUFFIX.length();
 
+  public static String buildBasic(String tableName, Object minTimeValue, Object maxTimeValue,
```

Comment:
These were lifted from the older class that is no longer used, but I will add the comments. These are used by thirdeye only, so we need to know what purpose they are being used, and I can add comments accordingly. Or, we should just move it to thirdeye code base, perhaps.